monitoring/config-generator/src/generate.js

```diff
@@ -164,362 +185,481 @@ function prometheusConfig (params) {
+          source_labels: [
+            '__address__',
+            '__meta_kubernetes_service_annotation_prometheus_io_port'
           ],
-          "separator": ";",
-          "regex": "([^:]+)(?::\\d+)?;(\\d+)",
-          "target_label": "__address__",
-          "replacement": "$1:$2",
-          "action": "replace"
+          separator: ';',
+          regex: '([^:]+)(?::\\d+)?;(\\d+)',
+          target_label: '__address__',
+          replacement: '$1:$2',
+          action: 'replace'
         },
         {
-          "separator": ";",
-          "regex": "__meta_kubernetes_service_label_(.+)",
-          "replacement": "$1",
-          "action": "labelmap"
+          separator: ';',
+          regex: '__meta_kubernetes_service_label_(.+)',
+          replacement: '$1',
+          action: 'labelmap'
         },
         {
-          "source_labels": [
-            "__meta_kubernetes_namespace"
-          ],
-          "separator": ";",
-          "regex": "(.*)",
-          "target_label": "kubernetes_namespace",
-          "replacement": "$1",
-          "action": "replace"
+          source_labels: ['__meta_kubernetes_namespace'],
+          separator: ';',
+          regex: '(.*)',
+          target_label: 'kubernetes_namespace',
+          replacement: '$1',
+          action: 'replace'
         },
         {
-          "source_labels": [
-            "__meta_kubernetes_service_name"
-          ],
-          "separator": ";",
-          "regex": "(.*)",
-          "target_label": "kubernetes_name",
-          "replacement": "$1",
-          "action": "replace"
+          source_labels: ['__meta_kubernetes_service_name'],
+          separator: ';',
+          regex: '(.*)',
+          target_label: 'kubernetes_name',
+          replacement: '$1',
+          action: 'replace'
         },
         {
-          "source_labels": [
-            "__meta_kubernetes_service_name"
-          ],
-          "separator": ";",
-          "regex": "(.*)",
-          "target_label": "livepeer_node_type",
-          "replacement": "$1",
-          "action": "replace"
+          source_labels: ['__meta_kubernetes_service_name'],
+          separator: ';',
+          regex: '(.*)',
+          target_label: 'livepeer_node_type',
+          replacement: '$1',
+          action: 'replace'
         }
       ]
     }
   ]
 }
+
+function getAlertManagerConfig (params) {
+  // global configuration
+  let global = {}
+
+  // The root route on which each incoming alert enters.
+  let route = {
+    group_by: ['alertname'],
+    group_wait: '2m',
+    group_interval: '30m',
+    // primary receiver
+    receiver: 'pagerduty',
+    // child routes
+    // can be used to send different severity notifications to different receivers
+    routes: [],
+  }
+
+  let receivers = []
+
+  // Inhibition rules allow to mute a set of alerts given that another alert is firing
+  let inhibit_rules = []
+  
+  inhibit_rules.push(
+    {
```

Comment:
Looks like this doesn't need to be indented.

Reply:
The repo doesn't have a linter (we should do this in every JS repo and have a company standard on which settings to use). I'll run beautify for now.

Reply:
There is a check if there is no service key provider: we return an empty object for the receiver.

Alertmanager will still process alerts and fire them but there will be no service to send them to I think. 

Haven't really tested out what happens when we omit it.

Reply:
Yeah that's a good idea. In this case, we can just be consistent with the indentation we're introducing with this set of changes and then a linter can be introduced separately.